pkgs/development/tools/xcbuild/platform.nix

```diff
@@ -0,0 +1,170 @@
+{ stdenv, sdk, writeText, platformName, xcbuild }:
+
+let
+
+  Info = {
+    CFBundleIdentifier = platformName;
+    Type = "Platform";
+    Name = "macosx";
+  };
+
+  Version = {
+    ProjectName = "OSXPlatformSupport";
+  };
+
+  Tools = [
+    {
+      Identifier = "com.apple.build-tools.nmedit";
+      Type = "Tool";
+      Name = "Nmedit";
+    }
+    {
+      Identifier = "com.apple.compilers.resource-copier";
+      Type = "Tool";
+      Name = "Resource Copier";
+    }
+    {
+      Identifier = "com.apple.compilers.yacc";
+      Type = "Tool";
+      Name = "Yacc";
+      InputFileTypes = [ "sourcecode.yacc" ];
+      ExecDescription = "Yacc $(InputFile)";
+    }
+    {
+      Identifier = "com.apple.compilers.lex";
```

Comment:
I think it generally looks good now, but did you get my comment earlier asking for some small commentary on where these things come from? Something like "This based mostly on https://github.com/facebook/xcbuild/tree/master/Specifications blah blah blah, but I had to change X for reasons Y and Z"